tests/test_new.py

```diff
@@ -0,0 +1,14 @@
+import playwright.sync_api
+from IPython.display import display
+
+
+def test_stream(ipywidgets_runner, page_session: playwright.sync_api.Page, assert_solara_snapshot):
+    def kernel_code():
+        from ipywebrtc import VideoStream
+
+        videoStream = VideoStream.from_file("./Big.Buck.Bunny.mp4")
+        display(videoStream)
+
+    ipywidgets_runner(kernel_code)
+    vid = page_session.locator(".video-stream").wait_for()
+    playwright.sync_api.expect(vid).to_be_visible()
```

Comment:
```suggestion
    page_session.locator(".video-stream").wait_for()
```

wait_for returns None, you could have done:
```
    vid = page_session.locator(".video-stream")
    vid.wait_for()
    playwright.sync_api.expect(vid).to_be_visible()
    ```
But that basically tests the same thing (see docstring of wait_for)